src/app/shared/layout/header.component.html

```diff
@@ -12,11 +12,16 @@
     View in Browser
     <mat-icon>link</mat-icon>
   </button>
-  <button *ngIf="auth.isAuthenticated() && needToShowReloadButton()" [disabled]="this.isReloadBtnDisabled"
+  <button *ngIf="auth.isAuthenticated() && needToShowReloadButton() && !this.isReloadBtnDisabled"
```

Comment:
*isReloadButtonDisabled

Reply:
does `needToShowReloadButton` return a boolean value? if yes rename to `isReloadButtonRequired`

Reply:
> does `needToShowReloadButton` return a boolean value? if yes rename to `isReloadButtonRequired`

hi, i am following the naming convention for displaying buttons in the header

![image](https://user-images.githubusercontent.com/6972112/60561080-e1bcc700-9d84-11e9-8dcb-228e2ae82de0.png)

Do you need me to change it all to is*ButtonRequired ?

Reply:
following the convention of naming functions that return a boolean value, I think it is better to rename them

Reply:
@RonakLakhotia I have made the necessary changes